### PR TITLE
chore: improve github actions

### PR DESCRIPTION
**Pull request by koushiro**
Signed-off-by: koushiro <koushiro.cqx@gmail.com>

### CODE REVIEW

**Comment by koushiro** [comment]
@vmx PTAL

**Comment by vmx** [comment]
As I expect the CI for rust-multibase being really similar likely even equal) to the rust-multiformats one, I'll do the review on https://github.com/multiformats/rust-multihash/pull/104 and once merged, we can then update this one accordingly.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,78 +1,93 @@
...
-      - uses: actions-rs/toolchain@v1
+      - name: Checkout Sources
+        uses: actions/checkout@v2
+
+      - name: Install Rust Toolchain
+        uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly
+          toolchain: stable
```
Comment (ia0) [inline .github/workflows/build.yml:63]: Stable cannot be used until 1.50 is released. When it is released, the new resolver must be used, see 
https://github.com/rust-lang/cargo/blob/master/src/doc/src/reference/resolver.md#resolver-versions for how to do it. I suggest using nightly for now.

Comment (koushiro) [reply .github/workflows/build.yml]: Thanks!

**Comment by koushiro** [comment]
@vmx PTAL again

**Review by vmx — APPROVED** [review]
As always, good work, thanks!

**Pull request merged**